more/free-programming-cheatsheets.md

```diff
@@ -239,6 +240,14 @@
 * [Nuxt.js Essentials Cheatsheet](https://www.vuemastery.com/pdf/Nuxtjs-Cheat-Sheet.pdf) - Vue Mastery (PDF)
 
 
+#### React-native
+
+* [Bridging native modules CheatSheet](https://gist.github.com/chourobin/f83f3b3a6fd2053fad29fff69524f91c) - Robin Chou (HTML)
+* [React native core components Cheatsheet](https://www.codecademy.com/learn/learn-react-native/modules/core-components-react-native/cheatsheet) - Codecademy (HTML)
+* [React native navigation Cheatsheet](https://www.codecademy.com/learn/learn-react-native/modules/navigation-react-native/cheatsheet) - Codeacademy (HTML)
+* [Styling components CheatSheet](https://www.codecademy.com/learn/learn-react-native/modules/styling-react-native/cheatsheet) - Codeacademy (HTML)
```

Comment:
The codecademy ones look like a book with all these concepts summary  than a cheatsheet. Would it be a value add here?

Reply:
@shrn01 ,
I agree with you, it may look like a book and gives summary of it. But it gives values here to my opinion.

Eg: check the react native documentation https://reactnative.dev/docs/textinput.

This is giving the entire information about TextInput in depth. What if we want to go with another component,
we may need to surf another link. 

These codeacemy cheatsheets are giving this kind of information to get started quickly.

What's suggestion regarding next steps.

Reply:
Cheatsheets are something that you can quickly reference if you need something, but this doesn't seem to be something like that. They are not something used to get started on something quickly

Reply:
What should be my next steps. Shall I do this in books secyion

Reply:
I have added these links in books section
https://github.com/EbookFoundation/free-programming-books/pull/8679

Kindly review it.